integration/test/performancetest/performance_test.go

```diff
@@ -1,36 +1,49 @@
-// go:build linux && integration
+//go:build linux && integration
 // +build linux,integration
 
 package performancetest
 
-import(
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"sync"
 	"testing"
 	"time"
-	"log"
-	"context"
+
 	"github.com/aws/amazon-cloudwatch-agent/integration/test"
 )
 
 const (
-	configPath = "resources/config.json"
-	configOutputPath = "/opt/aws/amazon-cloudwatch-agent/bin/config.json"
-	agentRuntimeMinutes = 5 //20min desired but 5mins for testing purposes 
-	DynamoDBDataBase = "CWAPerformanceMetrics"
-
+	configPath               = "resources/config.json"
+	configOutputPath         = "/opt/aws/amazon-cloudwatch-agent/bin/config.json"
+	agentRuntimeMinutes      = 5 //20 mins desired but 5 mins for testing purposes
+	DynamoDBDataBase         = "CWAPerformanceMetrics"
+	logOutputPath1           = "/tmp/test1.log"
+	logOutputPath2           = "/tmp/test2.log"
+	tpsMed 			 = 10
```

Comment:
One thing I don't think this addresses exactly is how we know which test corresponds to which use-case scenario/TPS/etc. Not something that needs to be tackled in this PR, but warrants discussion. 

For the `logOutputPath1` and logOutputPath2`, I think my other comment below addresses my concern with them, but I don't think we should have them hardcoded like this because we intend to have different numbers of log files monitored for performance testing.